### ISSUE TITLE
why multiply "number of examples in the current batch" when aggregating running loss

### ISSUE THREAD

**Issue by hopezh** [udacity/deep-learning-v2-pytorch#302]
May I ask why do we need to multiply "number of examples in the current batch", i.e. "*data.size(0)", when aggregating the running training loss, as shown below? 

There is no such operation in Udacity's tutorial introducing neural network. 

```
# update running training loss
train_loss += loss.item()*data.size(0)
```

**Comment by kHarshit** [comment]
Check this answer https://stackoverflow.com/q/61092523/6210807

**Comment by abhiojha8** [comment]
We suggest you refer to the Knowledge hub and reach out to your assigned mentor for user-specific issues. 
Thanks!
